homeworks/vlad.makedon_Vladmaked/homework_7/fibonacci.js

```diff
@@ -0,0 +1,13 @@
+function getRandomNumber(startOfRange, endOfRange) {
+    return (Math.floor(Math.random() * (endOfRange - startOfRange) + startOfRange));
+}
+
+function getFibonacciNumbers(rand = 55) {
+    const fiboArr = [0, 1];
+    for (let i = 2; fiboArr[i - 1] + fiboArr[i - 2] <= rand; i++) {
+        fiboArr.push(fiboArr[i - 1] + fiboArr[i - 2]);
+    }
```

Comment:
cool, but you could do it like `for (let i = 2, let fibo = fiboArr[i - 1] + fiboArr[i - 2]; fibo <= rand; i++) { fiboArr.push(fibo) }`

Reply:
I wanted to do this, but I did not succeed. I'll play with it again, thanks.